queue/migrations/00000000000000_create_jobs_table.up.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE jobs
+(
+    id           BIGINT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+    queue        TEXT      NOT NULL,
+    payload      JSONB     NOT NULL,
+    attempts     INTEGER   NOT NULL DEFAULT 0,
+    reserved_at  TIMESTAMP NOT NULL DEFAULT NULL,
+    available_at TIMESTAMP NOT NULL,
+    created_at   TIMESTAMP NOT NULL
+);
```

Comment:
The migration we currently use does not support multiple databases well. This migration was written for PostgreSQL.
Adding 4 different migration files seems a bit silly.

Reply:
But it seems we have to do it for now.